.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "2.6"
   - "2.7"
-script: make tests
+script: make tests-travis
```

Comment:
It not really services but setup the right user with sudo privileges.

1. Travis does not have the stanley user so some of the integration tests just won't work as they try to run an action.
2. Our own build system does not have sudo privileges for user running the test. Which means there is a test failure there as well. https://github.com/StackStorm/st2/pull/1062